crates/fuel-streams-core/src/transactions/types.rs

```diff
@@ -55,12 +55,14 @@ impl From<Transaction> for TransactionKind {
     }
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Default)]
 pub enum TransactionStatus {
     Failed,
     Submitted,
     SqueezedOut,
     Success,
+    #[default]
+    None,
```

Comment:
Is there a none status defined in fuel-core ? I thought its also either failed, submitted, squeezed out or successful ?

Reply:
Yeah, it's an Option<TransactionStatus> type. Unfortunately, our `IntoSubject` macro doesn't currently support Option<T> types. This might be a reason to want to revert to a simple trait (?). 

This is a normalized solution and it sort of makes sense since we want to ingest the status string, nothing else. What do you think?

Reply:
Oh, yes, sorry, Did not see its an option type. Then you need the default yes.